panel/lxqtpanel.cpp

```diff
@@ -230,6 +233,30 @@ LXQtPanel::LXQtPanel(const QString &configGroup, LXQt::Settings *settings, QWidg
 
     loadPlugins();
 
+    if(qGuiApp->nativeInterface<QNativeInterface::QWaylandApplication>())
+    {
+        // Create backing QWindow for LayerShellQt integration
+        create();
+
+        // Init Layer Shell (Must be done before showing widget)
+        mLayerWindow = LayerShellQt::Window::get(windowHandle());
```

Comment:
A nullity check for `windowHandle()`?

Reply:
I don't think `create()` will ever fail in this scenario but it doesn't hurt anyway.

Reply:
You're right. My request might be a little pedantic with `create()`. I thought more about `winId()`, which IMO is less "aggressive" (although I can't prove that). Anyway, nullity checks are always good :)